src/snippets/get-started/nuxt/Step3Route.js

```diff
@@ -0,0 +1,29 @@
+// @ts-check
+// @ts-expect-error
+import aj from "arcjet:client";
+
+export const GET = async ({ request }) => {
```

Comment:
This will need an update. Probably based on  https://arcjet-docs-git-quinn-nuxt-reference-arcjet.vercel.app/reference/nuxt#server-api-routes